src/internal.c

```diff
@@ -576,17 +579,32 @@ void CtxResourceFree(WOLFSSH_CTX* ctx)
 {
     WLOG(WS_LOG_DEBUG, "Entering CtxResourceFree()");
 
-    if (ctx->privateKey) {
-        ForceZero(ctx->privateKey, ctx->privateKeySz);
-        WFREE(ctx->privateKey, ctx->heap, DYNTYPE_PRIVKEY);
+    if (ctx->privateKeyCount > 0) {
```

Comment:
Do we need corresponding init code in CtxInit? e.g. settings the private key IDs array to unknown, etc.

Reply:
Check